web/controllers/v1/group_membership_controller.ex

```diff
@@ -18,6 +18,13 @@ defmodule Cog.V1.GroupMembershipController do
     render(conn, "show.json", group: group)
   end
 
+  def manage_group_users(conn, %{"users" => user_spec}=params) do
+    params
+    |> Map.put("members", %{"users" => user_spec})
+    |> Map.delete("users")
+    |> fn(param) -> manage_membership(conn, param) end.()
```

Comment:
Actually, `manage_memberships` is still used for granting roles to groups. I will double check that I haven't accidentally broken the tests to exercise that. I'll also pull the group related logic out of this function but I don't think there's more that needs to be done here to solve the problem this PR is trying to solve versus a general structural refactoring, which is out of scope.